com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.Prepass.cs

```diff
@@ -1052,10 +1055,14 @@ void DownsampleDepthForLowResTransparency(RenderGraph renderGraph, HDCamera hdCa
 
             using (var builder = renderGraph.AddRenderPass<DownsampleDepthForLowResPassData>("Downsample Depth Buffer for Low Res Transparency", out var passData, ProfilingSampler.Get(HDProfileId.DownsampleDepth)))
             {
-                // TODO: Add option to switch modes at runtime
-                if (m_Asset.currentPlatformRenderPipelineSettings.lowresTransparentSettings.checkerboardDepthBuffer)
+                if (hdCamera.isLowResScaleHalf)
                 {
                     m_DownsampleDepthMaterial.EnableKeyword("CHECKERBOARD_DOWNSAMPLE");
+                    m_DownsampleDepthMaterial.DisableKeyword("GREATER_THAN_HALF_DOWNSAMPLE");
+                }
+                else if (m_Asset.currentPlatformRenderPipelineSettings.lowresTransparentSettings.checkerboardDepthBuffer)
```

Comment:
Do we need the check on checkerboardDepthBuffer here? I think it's always true for now and if it wasn't the following is not doing checkerboard anyway.

Reply:
Cleaned this up.